src/makeRobotAccountant.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+  let result = [];
+
+  return function getSum(num) {
+    callCount++;
+    result.push(num);
+
+    return function next(nextNum) {
+      result.push(nextNum);
+
+      const sum = result.reduce((acc, curr) => acc + curr);
+
+      result = [];
+
+      if (callCount > 3 && callCount % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return sum;
+      }
```

Comment:
When you are returning inside an if statement, you don't need `else` afterwards
```suggestion
      if (callCount > 3 && callCount % 2 === 0) {
        return 'Bzzz... Error!';
      }
      
      return sum;
```